pyof/v0x01/controller2switch/packet_out.py

```diff
@@ -112,13 +112,17 @@ def _update_actions_len(self):
             self.actions_len = ListOfActions(self.actions).get_size()
 
     def _validate_in_port(self):
-        port = self.in_port
         valid = True
-        if isinstance(port, Port):
-            if port not in _VIRT_IN_PORTS:
+        if (isinstance(self.in_port, int) and
+                (self.in_port > 0 and self.in_port < Port.OFPP_MAX.value)):
```

Comment:
lines 116 to 118 treats valid ports that are not in ``Port`` class. the function returns so they don't get to be parsed to a instance of ``Port``.
lines 119 to 125 treats valid ports that are in the ``Port`` class.
Since OFPP_MAX is in the ``Port`` class, we should let it be parsed.

I left the ``.value`` there because it was already there. (commit 7c7faf7e)